airflow/providers/amazon/aws/operators/ec2.py

```diff
@@ -116,3 +116,139 @@ def execute(self, context: Context):
             target_state="stopped",
             check_interval=self.check_interval,
         )
+
+
+class EC2CreateInstanceOperator(BaseOperator):
+    """
+    Create and start an EC2 Instance using boto3
```

Comment:
Technically this could start multiple instances based on the `max_` and `min_count` args. Let's update this description to account for that behavior too.

Reply:
Good point, I'll update the description. Thanks!